File: beecrowd/Python/iniciante/1015.py
import math

def distancia(x1:float,x2:float,y1:float,y2:float):
    dist = math.sqrt(((x2 - x1)**2) + ((y2 - y1)**2))
    return print(f"{dist:.4f}")

x1,y1 = map(float, input().split())
x2,y2 = map(float, input().split())

resultado = distancia(x1,x2,y1,y2)

File: beecrowd/Cpp/iniciante/1015.cpp
#include <iostream>
#include <math.h>
#include <iomanip>

using namespace std;
// Distancia = √((x2 - x1)**2) + ((y2 -y1)**2)


int main() {
    double x1,x2,y1,y2;
    double distancia;

    cin >> x1 >> y1;
    cin >> x2 >> y2;
    cout << fixed << setprecision(4);

    distancia = sqrt(pow((x2 - x1),2) + pow((y2 - y1),2));
    cout << distancia;
    
    return 0;
}